lib/openstax/aws/stack.rb

```diff
@@ -308,6 +321,10 @@ def self.format_hash_as_stack_parameters(params={})
       end
     end
 
+    def self.format_hash_as_tag_parameters(tags)
+      tags.map{|tag| {key: tag.key, value: tag.value}}
```

Comment:
there are two tags.map ...one here and then one above on line 30.  can something be reused here?

Reply:
I think they are different - I could see a `Tags` object maybe that encapsulates this, but that would be the only method in such a class, so I'm ok with this.